pysqldb3/tests/test_csv_to_table.py

```diff
@@ -393,6 +417,59 @@ def test_bulk_csv_to_table_basic(self):
         # Cleanup
         db.drop_table(schema=pg_schema, table=create_table_name)
 
+    def test_bulk_csv_to_table_basic_blank_col(self):
+        # bulk_csv_to_table
+        db.query('drop table if exists {}.{}'.format(pg_schema, create_table_name))
+
+        fp = os.path.dirname(os.path.abspath(__file__)) + "\\test_data\\test8.csv"
```

Comment:
I saw that you use `DIR+` instead of `os.path.dirname(os.path.abspath(__file__))` in the `helpers.py` file. Is that something you wanted to change in this file too?

Reply:
yeah it should probably simplified to DIR

Reply:
resolved